stdlib/public/stubs/OptionalBridgingHelper.mm

```diff
@@ -0,0 +1,100 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#include <Foundation/Foundation.h>
```

Comment:
Nitpick: I think this counts as a system include, and so it should go at the bottom. Also, `#import` would be more appropriate. (ObjC headers typically don't have include guards.)